src/handlers/ens.ts

```diff
@@ -337,3 +322,48 @@ export const estimateENSRegisterSetRecordsAndNameGasLimit = async ({
   if (!gasLimit) return '0';
   return gasLimit;
 };
+
+export const formatRecordsForTransaction = (
+  records?: Records
+): ENSRegistrationRecords => {
+  const coinAddress = [] as { key: string; address: string }[];
+  const text = [] as { key: string; value: string }[];
+  let contentHash = null;
+  const ensAssociatedAddress = null;
+  records &&
+    Object.entries(records).forEach(([key, value]) => {
+      switch (key) {
+        case ENS_RECORDS.cover:
+        case ENS_RECORDS.twitter:
+        case ENS_RECORDS.displayName:
+        case ENS_RECORDS.email:
+        case ENS_RECORDS.url:
+        case ENS_RECORDS.avatar:
+        case ENS_RECORDS.description:
+        case ENS_RECORDS.notice:
+        case ENS_RECORDS.keywords:
+        case ENS_RECORDS.discord:
+        case ENS_RECORDS.github:
+        case ENS_RECORDS.reddit:
+        case ENS_RECORDS.instagram:
+        case ENS_RECORDS.snapchat:
+        case ENS_RECORDS.telegram:
+        case ENS_RECORDS.ensDelegate:
```

Comment:
How will this work for custom records?

Reply:
Maybe there should be a default case to add arbitrary records to the text records.

Reply:
yeah that wouldn't work, I'm not considering custom records for now